### ISSUE TITLE
 I try to train my own dataset，which has 16 points

### ISSUE THREAD

**Issue by xxy90** [Superlee506/Mask_RCNN#38]
Thank you for your reply.I try to use pytorch to train my own dataset ,but when i use the SGD optimizer with configs in config.py，after few steps, the loss became nan. When i try the Adam optimizer, the loss_keypoint drift and can't converge. Can you help me analyze the reason?
